### PR TITLE
fix(DB/creature): Auchenai Crypts pathing, formations and minor fixes

### PR DESCRIPTION
**Pull request by Gultask**
## Changes Proposed:
- Removes old pooling and replaces with id2
- Adds about 39 additional sniffed spawns, including cosmetic and bunny creatures
- Adjusts auras and addons
- Corrects movement speed for Auchenai Monk
- Implements sniffed waypoints and removes redundant pathing
- Implement proper formations
- Implements the cosmetic Soulpriests casting Blue Beam on a Cosmetic Raging Soul

## Issues Addressed:

- Closes 

## Tests Performed:

- Tested ingame
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ] Gameobjects (not in this PR)
- [ ] The "Phasing" creatures should perform events when their waypoints start/end. I couldn't manage to implement it.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nyeriah — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1668785413299016700.sql [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+-- Remove assistance from Reanimated Bones
+UPDATE `creature_template` SET `flags_extra`=`flags_extra`|33554432 WHERE `entry` = 18700;
+
+-- Condition for Blue Beam (32930)
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceGroup` = 1) AND (`SourceEntry` = 32930) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 18778) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(13, 1, 32930, 0, 0, 31, 0, 3, 18778, 0, 0, 0, 0, '', 'Blue Beam (32930) can only target Cosmetic Raging Soul (18778)');
+
+-- SAI for Blue Beam Soulpriests
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 18493;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1668785413299016700.sql:765]: This caught my attention, just put the 'SmartAI' to the guids that are being inserted a SAI below. (if it works)

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1668785413299016700.sql:765]: I can finally confirm that setting **only** the creature `ScriptName` to 'SmartAI' does not work

**Comment by Gultask** [comment]
Setting to WIP as I've found a way to implement the mechanic of phasing NPCs

**Comment by Gultask** [comment]
This uses waypoint_scripts instead of SAI. It works just as well if not better.

https://user-images.githubusercontent.com/100873791/205800536-95f7a2e7-b760-441b-87f8-16edd11ccbec.mp4

**Pull request merged**